src/DocumentFormat.OpenXml/OpenXmlDomReader.cs

```diff
@@ -89,7 +90,7 @@ public override ReadOnlyCollection<OpenXmlAttribute> Attributes
                 {
                     if (_emptyReadOnlyList == null)
                     {
-                        _emptyReadOnlyList = new ReadOnlyCollection<OpenXmlAttribute>(new List<OpenXmlAttribute>());
+                        _emptyReadOnlyList = Cached.ReadOnlyCollection<OpenXmlAttribute>();
```

Comment:
No need to store it in a field - you can just return the `Cached.ReadOnlyCollection`